UPGRADE-6.7.md

```diff
@@ -1,20 +1,86 @@
-* Parameter name `column` of `\Shopware\Core\Framework\Migration\MigrationStep::dropColumnIfExists` changed to `columnName` 
-* Parameter name `column` of `\Shopware\Core\Framework\Migration\MigrationStep::dropForeignKeyIfExists` changed to `foreignKeyName` 
-* Parameter name `index` of `\Shopware\Core\Framework\Migration\MigrationStep::dropIndexIfExists` changed to `indexName`
-## Removal of deprecated product review loading logic in Storefront
-* The service `\Shopware\Storefront\Page\Product\Review\ProductReviewLoader` was removed. Use `\Shopware\Core\Content\Product\SalesChannel\Review\AbstractProductReviewLoader` instead.
-* The event `\Shopware\Storefront\Page\Product\Review\ProductReviewsLoadedEvent` was removed. Use `\Shopware\Core\Content\Product\SalesChannel\Review\Event\ProductReviewsLoadedEvent` instead.
-* The hook `\Shopware\Storefront\Page\Product\Review\ProductReviewsWidgetLoadedHook` was removed. Use `\Shopware\Core\Content\Product\SalesChannel\Review\ProductReviewsWidgetLoadedHook` instead.
-* The struct `\Shopware\Storefront\Page\Product\Review\ReviewLoaderResult` was removed. Use `\Shopware\Core\Content\Product\SalesChannel\Review\ProductReviewResult` instead.
-## Native types for PHP class properties
-All PHP class properties now have a native type.
-If you have extended classes with properties, which didn't have a native type before, make sure you now add them as well.
+# 6.7.0.0 Upgrade Guide
+**NOTE:** All the breaking changes described here can be already opted in by activating the `v6.7.0.0` [feature flag](https://developer.shopware.com/docs/resources/references/adr/2022-01-20-feature-flags-for-major-versions.html#activating-the-flag) on previous versions.
+
+# Notable Changes
+
+# Webpack to vite migration for the administration
+We are switching the build system for our administration from webpack to vite. 
+This means that when your plugins depends on a custom `webpack.config.js` file, you'll need to migrate it to a `vite.config.js` file.
+**TBD how does that look like? Is there a docs page for that?**
+Additionally, this means that you will need to distribute a separate plugin version starting for 6.7, when you extend the administration to distribute the correct build files.
+For more information please take a look at the [docs](https://developer.shopware.com/docs/guides/plugins/plugins/administration/system-updates/vite.html).
+
+**Note:** This change can be activated separately with the `ADMIN_VITE` feature flag.
+
+# Vue.js Enhancements (full native vue 3 support)
+## Removal of Vue 2 compatibility layer
+The Vue 2 compatibility layer has been removed from the administration. This means that all components that still rely on Vue 2 features need to be updated.
+This ensures that our administration stays future-proof and we can make use of the most recent Vue 3 features.
+
+**What does that mean, can we link to a guide or something?**
```

Comment:
@shopware/product-ct-admin can you provide that?

Reply:
This migration build will be removed: https://v3-migration.vuejs.org/migration-build.html